packages/@uppy/informer/src/TransitionGroup.js

```diff
@@ -264,17 +264,10 @@ class TransitionGroup extends Component {
   render ({ childFactory, transitionLeave, transitionName, transitionAppear, transitionEnter, transitionLeaveTimeout, transitionEnterTimeout, transitionAppearTimeout, component, ...props }, { children }) {
     // TODO: we could get rid of the need for the wrapper node
```

Comment:
But we are not solving / getting rid of the todo?

Reply:
My change has nothing to do with the TODO, the TODO comes from https://github.com/developit/preact-transition-group/blob/f4ad5e3121729819aa3670edb79388036fb60ebc/src/TransitionGroup.js#L199, it was written by folks more informed than I am, I'm too afraid to touch it at this point.